modules/administration-guide/partials/proc_configuring_bitbucket_authentication.adoc

```diff
@@ -54,6 +54,10 @@ The main parts of the secret are:
 
 |===
 
+NOTE: Please note that encoding string into base64 format using *base64* tool on Linux machines most probably will
+add the newline character to the end of source string, so hash may become unusable as the authentication header value after decoding.
```

Comment:
I find "source string" confusing. Isn't it rather base64 encoded output string?

Reply:
sometimes it is both in encoded and decoded. You can try:

`echo 'NDc2MDc3NzczMTM4Ohe2i6Ji50ygk8CYeGZPCgh03a6d' | base64 >> out.txt`
there is newline in the file. and if you take a result and decode it back:
`echo 'TkRjMk1EYzNOemN6TVRNNE9oZTJpNkppNTB5Z2s4Q1llR1pQQ2doMDNhNmQK' | base64 -d >> out.txt`
you will see newline in decoded value and that's why header didn't work. And even `-w0` during decode not helps.
(use HEX mode to view file, `0A` is a `\n`)